binary-array-ld-test/src/main/resources/turtle/alias.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+@prefix dct: <http://purl.org/dc/terms/>
+@prefix owl: <http://www.w3.org/2002/07/owl#>
+@prefix skos: <http://www.w3.org/2004/02/skos/core#>
+@prefix bald: <https://www.opengis.net/def/binary-array-ld/>
+
+dct:publisher a owl:ObjectProperty ;
+    dct:identifier "dct_publisher" .
+
+rdfs:label a rdf:Property ;
+    dct:identifier "name" .
+
+skos:prefLabel a rdf:Property ;
+    dct:identifier "prefLabel" .
+
+bald:Organisation dct:identifier "binary-array-ld-org" .
```

Comment:
Could you explain why the bald:Organisation line does not include a declaration of ObjectProperty or Property? Is it implied?
Thanks

Reply:
This is a test case for [attribute value aliases](http://docs.opengeospatial.org/DRAFTS/19-002.html#_attribute_value_aliases), so it is the identifier of a generic resource rather than a property.